configs/dino/README.md

```diff
@@ -14,9 +14,10 @@ We present DINO (DETR with Improved deNoising anchOr boxes), a state-of-the-art
 
 ## Results and Models
 
-| Backbone |    Model    | Lr schd | box AP |                    Config                    |                                                                                                                                                  Download                                                                                                                                                  |
-| :------: | :---------: | :-----: | :----: | :------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-|   R-50   | DINO-4scale |   12e   |  49.0  | [config](./dino-4scale_r50_8xb2-12e_coco.py) | [model](https://download.openmmlab.com/mmdetection/v3.0/dino/dino-4scale_r50_8xb2-12e_coco/dino-4scale_r50_8xb2-12e_coco_20221202_182705-55b2bba2.pth) \| [log](https://download.openmmlab.com/mmdetection/v3.0/dino/dino-4scale_r50_8xb2-12e_coco/dino-4scale_r50_8xb2-12e_coco_20221202_182705.log.json) |
+| Backbone |    Model    | Lr schd | box AP |                     Config                      |                                                                                                                                                  Download                                                                                                                                                  |
+| :------: | :---------: | :-----: | :----: | :---------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|   R-50   | DINO-4scale |   12e   |  49.0  |  [config](./dino-4scale_r50_8xb2-12e_coco.py)   | [model](https://download.openmmlab.com/mmdetection/v3.0/dino/dino-4scale_r50_8xb2-12e_coco/dino-4scale_r50_8xb2-12e_coco_20221202_182705-55b2bba2.pth) \| [log](https://download.openmmlab.com/mmdetection/v3.0/dino/dino-4scale_r50_8xb2-12e_coco/dino-4scale_r50_8xb2-12e_coco_20221202_182705.log.json) |
+|  Swin-L  | DINO-5scale |   12e   |  57.2  | [config](./dino-5scale_swin-l_8xb2-12e_coco.py) |                                                                                                                                          [model](<>) \| [log](<>)                                                                                                                                          |
```

Comment:
We uploaded the model weight to the cloud.
You can update the link now.
Model: https://download.openmmlab.com/mmdetection/v3.0/dino/dino-5scale_swin-l_8xb2-12e_coco/dino-5scale_swin-l_8xb2-12e_coco_20230228_072924-a654145f.pth
Log: https://download.openmmlab.com/mmdetection/v3.0/dino/dino-5scale_swin-l_8xb2-12e_coco/dino-5scale_swin-l_8xb2-12e_coco_20230228_072924.log